CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Improve DCAT harvest of mime type [#2857](https://github.com/opendatateam/udata/pull/2857)
 - Don't crash on files not found when purging resources [2858](https://github.com/opendatateam/udata/pull/2858)
+- Expose dataset's created_at_internal date in api endpoints [#2862](https://github.com/opendatateam/udata/pull/2862)
```

Comment:
I guess we want the the same modification for last_modified?
And I think this applies at the dataset and the resource model, right?